<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalRecordDialog</class>
 <widget class="QDialog" name="MedicalRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>病历信息</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>病历信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 16pt; font-weight: bold; color: #8B4513; margin-bottom: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="patientInfoGroup">
     <property name="title">
      <string>患者信息</string>
     </property>
     <property name="styleSheet">
      <string>QGroupBox { font-weight: bold; color: #8B4513; }</string>
     </property>
     <layout class="QFormLayout" name="patientFormLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="patientLabel">
        <property name="text">
         <string>患者:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="patientComboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>32</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QFrame" name="patientDetailsFrame">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Sunken</enum>
        </property>
        <property name="styleSheet">
         <string>QFrame { background-color: #FFF8DC; border: 1px solid #D2B48C; border-radius: 6px; padding: 8px; }</string>
        </property>
        <layout class="QGridLayout" name="patientDetailsLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="patientNameLabel">
           <property name="text">
            <string>姓名:</string>
           </property>
           <property name="styleSheet">
            <string>font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="patientNameValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="patientAgeLabel">
           <property name="text">
            <string>年龄:</string>
           </property>
           <property name="styleSheet">
            <string>font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="patientAgeValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="patientGenderLabel">
           <property name="text">
            <string>性别:</string>
           </property>
           <property name="styleSheet">
            <string>font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="patientGenderValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="patientPhoneLabel">
           <property name="text">
            <string>电话:</string>
           </property>
           <property name="styleSheet">
            <string>font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="patientPhoneValue">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="medicalRecordGroup">
     <property name="title">
      <string>病历信息</string>
     </property>
     <property name="styleSheet">
      <string>QGroupBox { font-weight: bold; color: #8B4513; }</string>
     </property>
     <layout class="QFormLayout" name="medicalFormLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="diagnosisLabel">
        <property name="text">
         <string>诊断: *</string>
        </property>
        <property name="styleSheet">
         <string>color: #8B4513;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="diagnosisEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>32</height>
         </size>
        </property>
        <property name="maxLength">
         <number>500</number>
        </property>
        <property name="placeholderText">
         <string>请输入诊断信息（必填，最多500字符）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="symptomsLabel">
        <property name="text">
         <string>症状:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="styleSheet">
         <string>color: #8B4513;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="symptomsEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入症状描述（最多1000字符）</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="treatmentLabel">
        <property name="text">
         <string>治疗方案:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="styleSheet">
         <string>color: #8B4513;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="treatmentEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入治疗方案（最多1000字符）</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="doctorNameLabel">
        <property name="text">
         <string>医生姓名: *</string>
        </property>
        <property name="styleSheet">
         <string>color: #8B4513;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="doctorNameEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>32</height>
         </size>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
        <property name="placeholderText">
         <string>请输入医生姓名（必填，最多100字符）</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="notesLabel">
        <property name="text">
         <string>备注:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="styleSheet">
         <string>color: #8B4513;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="notesEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入备注信息（最多2000字符）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>color: red; font-weight: bold;</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>